### PR TITLE
Initial commit of express app 

### PR DESCRIPTION
**Pull request by sufikaur**
New files added as of June 26

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,19 @@
+{
+  "name": "test",
```
Comment (tsingh777) [inline package.json:2]: Update this `SikhiFM API`

**Comment by tsingh777** [comment]
Update the PR title to be more meaningful.

File: package.json [hunk 2]
```diff
@@ -0,0 +1,19 @@
+{
+  "name": "SikhiFM API",
+  "version": "1.0.0",
+  "description": "test express",
+  "main": "index.js",
```
Comment (tsingh777) [inline package.json:5]: change this to `app.js` so the `npm start` command picks up the correct file.

File: .gitignore [hunk 3]
```diff
@@ -0,0 +1 @@
+node_modules
```
Comment (tsingh777) [inline .gitignore:1]: Lets use https://www.toptal.com/developers/gitignore/api/node. you can copy the gitignore this generates

File: app.js [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+
+const app = express();
+const port = process.env.NODE_ENV === 'development' ? '3005' : '3004';
+
+// database
+app.locals.pool = createPool(config.mysql);
+app.locals.pool.add('local', config.mysql0);
+// app.locals.pool.add('db1', config.mysql1);
+// app.locals.pool.add('db2', config.mysql2);
+// app.locals.pool.add('db3', config.mysql3);
```
Comment (tsingh777) [inline app.js:24]: remove the pooling for now. we can come back to that.

File: api/controllers/category.js [hunk 5]
```diff
@@ -0,0 +1,6 @@
+// make exports
```
Comment (tsingh777) [inline api/controllers/category.js:1]: lets remove empty files for now.

File: api/controllers/collection.js [hunk 6]
```diff
@@ -0,0 +1,12 @@
+
```
Comment (tsingh777) [inline api/controllers/collection.js:1]: lets remove empty files for now.

File: api/controllers/shabad.js [hunk 7]
```diff
@@ -0,0 +1,7 @@
+// imports
```
Comment (tsingh777) [inline api/controllers/shabad.js:1]: lets remove empty files for now.

File: api/config/index.js [hunk 8]
```diff
@@ -0,0 +1,27 @@
+const express = require('express')
+const app = express()
+const port = 3000
+
+app.get('/', (req, res) => res.send('Hello World!'))
+
+// GET method route
+app.get('/hello/:hello', function (req, res) {
```
Comment (tsingh777) [inline api/config/index.js:8]: this should be in routes

File: api/config/index.js [hunk 9]
```diff
@@ -0,0 +1,27 @@
+const express = require('express')
+const app = express()
+const port = 3000
+
+app.get('/', (req, res) => res.send('Hello World!'))
```
Comment (tsingh777) [inline api/config/index.js:5]: this should not be needed since we are already responding to `'/'`

File: api/config/index.js [hunk 10]
```diff
@@ -0,0 +1,27 @@
...
+
+// GET method route
+app.get('/hello/:hello', function (req, res) {
+    res.send('GET request to the homepage')
+  })
+  
+
+app.listen(port, () => console.log(`Example app listening at http://localhost:${port}`))
```
Comment (tsingh777) [inline api/config/index.js:13]: this should not be needed since we doing this in app.js.

File: package.json [hunk 11]
```diff
@@ -0,0 +1,19 @@
+{
+  "name": "SikhiFM API",
+  "version": "1.0.0",
+  "description": "test express",
```
Comment (tsingh777) [inline package.json:4]: Update the description.

File: app.js (hunk 4 above)
Comment (sufikaur) [reply app.js:24]: The app doesnt seem to run without the first line

**Pull request merged**